changelog.yaml

```diff
@@ -8508,3 +8508,8 @@
     added:
     - Biden NIIT label and description change.
   date: 2024-10-17 19:59:12
+- bump: minor
```

Comment:
add to `changelog_entry.yaml` not to `changelog.yaml` - please see other closed PR's for reference